tmail-backend/events/src/main/scala/com/linagora/tmail/events/TmailEvent.scala

```diff
@@ -0,0 +1,22 @@
+package com.linagora.tmail.events
+
+import org.apache.james.core.{MailAddress, Username}
+import org.apache.james.events.Event
+import org.apache.james.events.Event.EventId
+
+trait TmailEvent extends Event {
+
+}
+
+trait TmailContactUserEvent extends TmailEvent {
+
+}
+
+case class TmailContactUserAddedEvent(eventId: EventId, username: Username, contact: MailAddress, firstname: String, lastname: String) extends TmailContactUserEvent {
```

Comment:
IMO we should reuse the `ContactFields` POJO from https://github.com/linagora/tmail-backend/pull/384